Source/Particles/AddPlasmaUtilities.H

```diff
@@ -22,6 +22,29 @@
 #include <AMReX_REAL.H>
 #include <AMReX_RealBox.H>
 
+struct PDim3 {
```

Comment:
I had to move this object from `PhysicalParticleContainer.cpp` to `AddPlasmaUtilities.H` as part of this PR.